simde/arm/neon/mul_lane.h

```diff
@@ -386,7 +386,7 @@ simde_vmul_laneq_s32(simde_int32x2_t a, simde_int32x4_t b, const int lane)
   #else
     SIMDE_VECTORIZE
     for (size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
-      r_.values[i] = a_.values[i] * b_.values[lane];
+      r_.values[i] = (uint32_t)a_.values[i] * (uint32_t)b_.values[lane];
```

Comment:
Thank you, but this generated clang errors:

https://github.com/simd-everywhere/simde/actions/runs/15861220962/job/44718601505?pr=1296#step:10:666
```suggestion
      r_.values[i] = (int32_t)((uint32_t)a_.values[i] * (uint32_t)b_.values[lane]);
```